.travis.yml

```diff
@@ -4,9 +4,6 @@ language: cpp
 os:
   - linux
   - osx
-compiler:
-  - clang
```

Comment:
@tamird seems TEST_GROUP 1&2 is taking an hour to run. Mind remove them and add `make all` for osx?